### ISSUE TITLE
Warning: Calling 'depends_on :autoconf' is deprecated!

### ISSUE THREAD

**Issue by N4tr0n** [railwaycat/homebrew-emacsmacport#72]
I got this warning when running `brew update` and I figured I should report it like the message asks.
```
$ brew update
Updated Homebrew from b999edb to a5a930c.
Warning: Calling 'depends_on :autoconf' is deprecated!
Use 'depends_on "autoconf" => :build' instead.
/usr/local/Library/Taps/railwaycat/homebrew-emacsmacport/Formula/emacs-mac.rb:10:in `<class:EmacsMac>'
Please report this to the railwaycat/emacsmacport tap!

Warning: Calling 'depends_on :automake' is deprecated!
Use 'depends_on "automake" => :build' instead.
/usr/local/Library/Taps/railwaycat/homebrew-emacsmacport/Formula/emacs-mac.rb:11:in `<class:EmacsMac>'
Please report this to the railwaycat/emacsmacport tap!

Updated 3 taps (caskroom/cask, homebrew/core, railwaycat/emacsmacport).
```

**Comment by railwaycat** [comment]
This issue should fixed after 0a0f78d, could you have another try of `brew update -v` to see how it goes? Thanks!

#70

**Comment by N4tr0n** [comment]
Just ran `brew update -v` and I didn't see any warnings.
